proposals/NNNN-localization.md

```diff
@@ -0,0 +1,107 @@
+Configuration config = new Configuration(resources.getConfiguration());
+config.setLocale(locale);
+ContextWrapper wrapper = new ContextWrapper(context);
+this.context = wrapper.createConfigurationContext(config);
+```
+
+This proposal is using an API called [createConfigurationContext()](https://developer.android.com/reference/android/content/ContextWrapper.html#createConfigurationContext(android.content.res.Configuration)) which was added at API level 17 (Android 4.2 Jelly Beans MR1). This method creates a new context separated from the apps configuration which allows the app to access resources of another language.
+
+The downside is the API level (see [Android dashboards](https://developer.android.com/about/dashboards/index.html)). As it is expected this number to become lower overtime and as it already reached a low number it is proposed to just use the phone language instead.
+
+```java
+this.context = context.getApplicationContext(); // just use the original context
+```
+
+This class should be added to the Android SDL library and be used when Android gets a lifecycle manager like structure. This can make the class available to app developers in an early phase before having a lifecycle manager ready.
+
+#### iOS
+
+For iOS the localization class loads language bundles based on a specified locale (language, script, region). iOS prioritizes script rather than the region (see [Hant](http://www.unicode.org/cldr/charts/latest/summary/root.html#71) and [Hans](http://www.unicode.org/cldr/charts/latest/summary/root.html#70)). That said: the localization class supports scripts.
+
+At the end the localization class loads a list of bundles including localizable files. The localization class provide access to those files with `-stringForKey: ...` which include support of format and plural rules.
+
+This localization class should be used by the SDLLifecycleManager. An instance should be created right after the app got connected. This instance should be made accessible to the developer by a property e.g. `@property ... SDLLocalization *localization;`
+
+## Potential downside
+
+No downside identified for iOS. Downside of Android mentioned above.
+
+## Impact on existing code
+
+This change whould not affect any existing code as it's adding classes and a single property to the lifecycle manager.
+
+## Alternatives considered
+
+**Change language configuration of the entire app** would be an easy approach (very few lines of code) but causes the UI of the app on the phone to change to the SDL language. This was dropped as it is too agressive.
+
+**Create custom localization files** separate from the existing and known ones. This would make localization is independent of the Android or iOS API  but it's much more effort to follow CLDR specifications especially when it comes to plural rules (which are very helpful for natural voice output). Furthermore the app developer has to do translations outside of the known environment which may not be accepted.
```

Comment:
Please change the beginning of the first sentence to "This would make localization independent..."